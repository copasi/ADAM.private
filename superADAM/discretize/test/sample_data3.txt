3.2185710377470755   3.2185710377470755   -0.001647374853474741
3.2185710377470755   2.578856652174989   -0.0005838353213798433
3.2185710377470755   2.252993554768716   -0.0001958074187537289
3.2185710377470755   2.1032649423594543   -0.000056197791256863725
3.2185710377470755   2.0378573990797895   -0.00000506141704481992
3.2185710377470755   2.009781163915104   0.000013962720917306831
3.2185710377470755   1.9979436406936046   0.000020967783852639085
3.2185710377470755   1.992941828704291   0.00002364704946903226
3.2185710377470755   1.9913615846144574   0.000024286012479213278
3.2185710377470755   1.9905653816762108   0.000024626899577237192 

