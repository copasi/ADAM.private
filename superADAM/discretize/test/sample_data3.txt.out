0

2	0	1	
2	1	1	
2	1	1	
2	1	1	
2	1	1	
2	2	1	
1	2	1	
1	2	1	
1	2	1	
1	2	1	
0	2	0	
One or more of your variables does not take on enough values to obtained the appropriate field cardinality.